lib/xnvme_be_macos_dev.c

```diff
@@ -30,8 +30,8 @@ _get_ioservice_device(char *devname)
 	CFMutableDictionaryRef matching_dictionary;
 	io_object_t ioservice_device;
 
-	matching_dictionary = IOBSDNameMatching(kIOMasterPortDefault, 0, basename(devname));
-	ioservice_device = IOServiceGetMatchingService(kIOMasterPortDefault, matching_dictionary);
+	matching_dictionary = IOBSDNameMatching(kIOMainPortDefault, 0, basename(devname));
```

Comment:
With the environment variable set, then I think this should be dropped, as the toolchain for 11 would then not understand the "new" ``kIOMainPortDefault``.